scripts/release.js

```diff
@@ -47,6 +47,7 @@ const deployDemo = (version) => {
   updateDependency(`amazon-chime-sdk-js@${latestNPMJSSdkVersion}`);
   process.chdir(path.join(__dirname, '../../amazon-chime-sdk/apps/meeting'));
   updateDependency('amazon-chime-sdk-component-library-react', `../../../amazon-chime-sdk-component-library-react/amazon-chime-sdk-component-library-react-${version}.tgz`);
+  spawnOrFail('rm', [`-rf ../../../amazon-chime-sdk-component-library-react/amazon-chime-sdk-component-library-react-${version}.tgz`]);
```

Comment:
Should we just add this it in gitignore so we never commit any tar files?

Reply:
Sounds like a more reliable way. Let me close this one and open a new PR.

Reply:
Done